BookingService/BookingService.Domain/User.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNet.Identity.EntityFramework;
+
+namespace BookingService.Domain
+{
+    public class User : IdentityUser
+    {
+        public string Role { get; set; }
+    }
+}

```

Comment:
U cannot specify role here. AspNetIdentity framework allows u to manage roles out of the box. Decompile IdentityUser and u will see that it has list of `IdentityUserRole`. 1 user can have many roles, 1 role can have many users. it looks like
User 1 --< M UserRole M >--- Role